### PR TITLE
Create 1g3cN4JRE3PKmcCxL0OSN3

### PR DESCRIPTION
**Pull request by Censeii**

### CODE REVIEW

**Comment by maciejpedzich** [comment]
Note to @mackorone regarding the three-hour-long response delay

Looks like I missed a breaking change when updating the bot's dependencies. The web server it spins up now expects webhook payloads to be delivered via a `POST /api/github/webhooks` request instead of `POST /`. The latter now responds with a 404, so I had to manually change the delivery path via the bot's configuration page and resend the last payload.

TL;DR - fixed breaking change; bot should be faster to respond
